helm/fdb-operator/templates/manager/statefulset.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: controller-manager
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "chart.labels" . | nindent 4 }}
+spec:
+  selector:
+    matchLabels:
+      {{- include "chart.selectorLabels" . | nindent 6 }}
+  serviceName: controller-manager-service
+  template:
+    metadata:
+      labels:
+        {{- include "chart.labels" . | nindent 8 }}
+    spec:
+      volumes:
+        - name: cert
+          secret:
+            defaultMode: 420
+            secretName: {{ .Values.webhook.secret.name }}
+        - name: fdb-certs
```

Comment:
The `fdb-certs` secret is part of the local development environment, but I don't think we want it in the helm chart.

Reply:
if you see the statefulset that gets created for the operator, manifest of that statefulset has this secret mounted, so I just made that more manageable but having the name in the values.yaml file, but it was already in the statefulset's manifest.

Reply:
We should base this on the latest sample deployment YAML (https://github.com/FoundationDB/fdb-kubernetes-operator/blob/master/config/samples/deployment.yaml) after the Kubebuilder 2 upgrade.